integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -1227,6 +1227,138 @@ msgstr "Monatlich"
 msgid "Yearly"
 msgstr "Jährlich"
 
+#: cms/constants/language_color.py
+msgid "Mellow apricot"
+msgstr "Milde Aprikose"
+
+#: cms/constants/language_color.py cms/constants/poicategory.py
+msgid "Forest green"
+msgstr "Waldgrün"
+
+#: cms/constants/language_color.py
+msgid "Rose"
+msgstr "Rosa"
+
+#: cms/constants/language_color.py
+msgid "Tropical violet"
+msgstr "Tropisches Veilchen"
+
+#: cms/constants/language_color.py cms/constants/poicategory.py
+msgid "Red"
+msgstr "Rot"
+
+#: cms/constants/language_color.py cms/constants/poicategory.py
+msgid "Orange"
+msgstr "Orange"
+
+#: cms/constants/language_color.py cms/constants/poicategory.py
+msgid "Dark blue"
+msgstr "Dunkelblau"
+
+#: cms/constants/language_color.py
+msgid "Green blue"
+msgstr "Grün-blau"
+
+#: cms/constants/language_color.py cms/constants/poicategory.py
+msgid "Yellow"
+msgstr "Gelb"
+
+#: cms/constants/language_color.py
+msgid "Teal"
+msgstr "Türkisblau"
+
+#: cms/constants/language_color.py
+msgid "Arctic"
+msgstr "Arktisch"
+
+#: cms/constants/language_color.py
+msgid "Azure"
+msgstr "Azurblau"
+
+#: cms/constants/language_color.py
+msgid "Pacific blue"
+msgstr "Pazifikblau"
+
+#: cms/constants/language_color.py
+msgid "Orange red"
+msgstr "Orangerot"
+
+#: cms/constants/language_color.py
+msgid "Light green"
+msgstr "Hellgrün"
+
+#: cms/constants/language_color.py
+msgid "Violet"
+msgstr "Violett"
+
+#: cms/constants/language_color.py
+msgid "Lime"
+msgstr "Limette"
+
+#: cms/constants/language_color.py cms/constants/poicategory.py
+msgid "Lavender"
+msgstr "Lavendel"
+
+#: cms/constants/language_color.py
+msgid "Brown"
+msgstr "Braun"
+
+#: cms/constants/language_color.py
+msgid "Pink orange"
+msgstr "Rosa-orange"
```

Comment:
Not found in Duden, probably rosaorange? But it's a newly created word as far as Duden is concerned :sweat_smile: